In setup-runner, change the cp $TFO_MAIN_MODULE_ADDONS to be failsafe

Cytrian opened the pull request:
In case of an empty directory $TFO_MAIN_MODULE_ADDONS the cp command fails silently.
In this case, the whole setup script ends without an error message.

We had that case with an Terraform resource without a customBackend.

I'm also not sure about the -i switch to cp, does not make any sense in a script.

Commit:
In setup-runner, change the cp $TFO_MAIN_MODULE_ADDONS to be failsafe

In case of an empty directory $TFO_MAIN_MODULE_ADDONS the cp command fails silently.
In this case, the whole setup script ends without an error message.

We had that case with an Terraform resource without a customBackend.

I'm also not sure about the -i switch to cp, does not make any sense in a script.

isaaguilar commented on terraform-runner/setup.sh:
```diff
@@ -59,7 +59,8 @@ fi
 mkdir -p $TFO_MAIN_MODULE
-false |  cp -iLr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null
+
+cp -Lr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null | true
```
The combination of `false | cp -iLr ` makes sure to not overwrite any files in the target that already exist with the same name by filling in the prompt with the piped input `false`.  

I think making it failsafe is fine as I have no strong opinion against it. But I don't want to remove the "no overwrite" behavior... although now that I think about it, the "no overwrite" behavior was before the generational directory structure. Hm... Well I'll just suggest the simplest change which is to keep everything and add the pipe true.

```suggestion
false |  cp -iLr "$TFO_MAIN_MODULE_ADDONS"/* "$TFO_MAIN_MODULE" 2>/dev/null | true
```

Cytrian replied:
Ah, now I understand the hack. Okay, but, then we shouldn't pipe into "true".
"|| true" is the correct way.

Cytrian committed:
re-add the overwrite-prevention

Re-add the overwrite-prevention ("false | cp -i")
Although "false | true" does actually work to prevent an immediate exit, "false || true" is the correct way.
